symbols.yaml

```diff
@@ -267,7 +267,7 @@ punctuator:
 #單位
     '/dw': [ Å, ℃, ％, ‰, ‱, °, ℉, ㏃, ㏆, ㎈, ㏄, ㏅, ㎝, ㎠, ㎤, ㏈, ㎗, ㎙, ㎓, ㎬, ㏉, ㏊, ㏋, ㎐, ㏌, ㎄, ㎅, ㎉, ㎏, ㎑, ㏍, ㎘, ㎞, ㏎, ㎢, ㎦, ㎪, ㏏, ㎸, ㎾, ㏀, ㏐, ㏓, ㎧, ㎨, ㎡, ㎥, ㎃, ㏔, ㎆, ㎎, ㎒, ㏕, ㎖, ㎜, ㎟, ㎣, ㏖, ㎫, ㎳, ㎷, ㎹, ㎽, ㎿, ㏁, ㎁, ㎋, ㎚, ㎱, ㎵, ㎻, ㏘, ㎩, ㎀, ㎊, ㏗, ㏙, ㏚, ㎰, ㎴, ㎺, ㎭, ㎮, ㎯, ㏛, ㏜, ㎔, ㏝, ㎂, ㎌, ㎍, ㎕, ㎛, ㎲, ㎶, ㎼ ]
 #貨幣
-    '/hb': [ ￥, ¥, ¤, ¢, ￠, ＄, $, €, ￡, £, ₳, ৲, ৳, ฿, ₿, ₠, ₵, ₡, ₢, ₫, ₯, ₣, ૱, ₲, ₴, ₹, ៛, ₭, ₾, ₤, ₶, ₼, ₥, ₦, ₪, ₻, ₰, ₧, ₱, ₽, ₨, ﷼, ௹, ₷, ₸, ₮, ₺, ₩, ￦ ]
+    '/hb': [ ￥, ¥, ¤, ¢, ￠, ＄, $, €, ￡, £, ₳, ৲, ৳, ฿, ₿, ₠, ₵, ₡, ₢, ₫, ₯, ₣, ૱, ₲, ₴, ₹, ៛, ₭, ₾, ₤, ₶, ₼, ₥, ₦, ₪, ₻, ₰, ₧, ₱, ₽, ₨, ⁦﷼⁩, ௹, ₷, ₸, ₮, ₺, ₩, ￦ ]
```

Comment:
Why add left to right annotation here? If this is Arabic, it will be used with Arabic script.

Reply:
But the overall content is LTR. You can try it. RTL without isolation will mess up the layout of that candidate (up until line feed). So you will see the candidate on the left and number label on the right.

Reply:
Then users just need to make sure the Arabic numerals are inside the two isolation marks. These marks aren't combining marks, nor do they form multi-character symbols that cannot be separated. It's just like inserting inside a pair of parentheses.
I don't think anyone would appreciate the following situation:
![Screenshot 2023-02-18 at 03 25 23 (2)](https://user-images.githubusercontent.com/14979243/219827333-adfbbfa1-d695-4df1-b81c-3b64c5147d15.png)

Reply:
Then remove it from the list.

Reply:
LTR isolation is the legit solution and recommended by unicode. Why are you so destructive?

Reply:
I'm not destruction, it's you that is so resistant.